packages/docs/src/03-Usage.svench/20-knobs.md

```diff
@@ -0,0 +1,105 @@
+<script>
+   import { View } from 'svench';
+</script>
+
+<svench:options title="Knobs" />
+
+# Knobs
+
+Knobs are used to interact with the View components.
+
+## Defining and using knobs
+
+Knobs are defined for a `View` via the `knobs` prop, and their current values
+are retrieved via `let:knobs`.
+
+```svelte
+<View name="defining and using a knob, simple" knobs={{ someProp: "my default value" }} let:knobs>
+
+    <p>someProp: {knobs.someProp}</p>
+
+</View>
+
+<!-- equivalent, using deconstructors -->
+
+<View name="defining and using a knob, using deconstructor" 
+      knobs={{ someProp: "my default value" }} 
+      let:knobs={{someProp}}>
+
+    <p>someProp: {someProp}</p>
+
+</View>
+```
+
+<View name="defining and using a knob, simple" knobs={{ someProp: "my default value" }} let:knobs>
+
+    <p>someProp: {knobs.someProp}</p>
+
+</View>
+
+<View name="defining and using a knob, using deconstructor" knobs={{ someProp: "my default value" }} let:knobs={{someProp}}>
+
+    <p>someProp: {someProp}</p>
+
+</View>
+
+## Knobs passed as plain objects
+
+Knobs can be passed to the views as an object, where the keys are the knob
+names and the values are their defaultvalues:
+
+```svelte
+<View name="knobs as objects" 
+      knobs={{ firstProp: "hello world", secondProp: 123 }} 
```

Comment:
This is only the shortcut notation, knobs defined as objects can have full options too:

```js
knobs={{
  shortcut: false,
  optioned: { type: "number", default: 123 },
}}
```

I guess I added array mainly as a mean of controlling the order in which knob inputs are rendered (which should probably be explained in the docs).

Reply:
Both long-form and array justification added to the docs!